plasma_framework/contracts/src/vaults/Erc20Vault.sol

```diff
@@ -16,6 +16,13 @@ contract Erc20Vault is Vault {
         uint256 amount
     );
 
+    event DepositCreated(
```

Comment:
Why not just "Deposit"?

Reply:
This `DepositCreated` is a legacy from `RootChain`, but I do agree that event `Deposit` or `Deposited` would be probably better.